src/tests/system/tests/test_ldap_extra_attrs.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ticket(gh=4153, bz=1362023)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_ldap_extra_attrs__contains_mail(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD fails to start when ldap_user_extra_attrs contains mail
```

Comment:
SSSD starts correctly when ...

Reply:
Because the expected result is that SSSD starts. The original title suggests that the expected result is that SSSD fails to start.